<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tiny6410Dialog</class>
 <widget class="QDialog" name="tiny6410Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>tiny6410Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="ledButton">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>48</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string extracomment="tiny6410 led test">LED</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyButton">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>48</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string extracomment="tiny6410 led test">KEY</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pwmButton">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>48</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string extracomment="tiny6410 led test">PWM</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="adcButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string extracomment="tiny6410 led test">ADC</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
